vividus-http-client/src/main/java/org/vividus/http/client/HttpClient.java

```diff
@@ -82,7 +78,6 @@ public HttpResponse execute(HttpUriRequest request) throws IOException
     @Override
     public HttpResponse execute(HttpUriRequest request, HttpContext context) throws IOException
     {
-        LOGGER.info("{}", request);
```

Comment:
Why the same approach as skipResponseEntity doesn't work in this case?
What should I do if I want to enable console logging in the other place?

Reply:
- I think because skip response entity directly affects the response that we return, whereas logging doesn't.
- I also have doubts about introducing new boolean switch because we "add" one more logger disabling mechanism in addition to one provided by logging itself
- I think you meant allure logging, you will have to configure one more logger and register new http client bean
- anyway I'll consider this since your suggestion seems more concise

Reply:
Introducing of new property won't help since we still the logging to go into console and file appenders, so it requires configuring a new logger

Reply:
Yeah, in simple words can we design it to be applicable by introducing changes in the single place, not the two xmls

Reply:
why two xmls? once we have such real case we will definitely try to reuse it :)